### ISSUE TITLE
South facing rope(s

### ISSUE THREAD

**Issue by stormhalter** [jkachhad/Stormhalter#737]
cannot be climbed using mouse) These ropes, when doubleclicked, return 'You can't do that here'. Specific instance in the rope up to Por's pub. [Reported by korazail]

---
[View on Discord](https://discord.com/channels/1056432310632841276/1530620982488858775/1530620982488858775)

---
[View on Discord](https://discord.com/channels/577503035526348810/1532158505941995660/1532158505941995660)

**Comment by sproatba** [comment]
I believe the action on these should be action 2 instead of 5. They are not exposed to us so we can't fix.  Bumped to high prio since theoretically this is a quick fix.  If I'm wrong feel free to demote it.

<terrain id="512" lom="335" action="5" tif="427,637">
    <sprite order="6">
      <texture>Bitmaps\004</texture>
      <source>(700,2100,100,100)</source>
      <offset>(-1,12)</offset>
    </sprite>
    <average>(193,120,77,255)</average>
  </terrain>
  <terrain id="513" lom="336" action="5" tif="428">
    <sprite order="3">
      <texture>Bitmaps\004</texture>
      <source>(800,2100,100,100)</source>
    </sprite>
    <average>(191,118,76,255)</average>
  </terrain>
  <terrain id="514" lom="337" action="5" tif="429,526">
    <sprite order="7">
      <texture>Bitmaps\004</texture>
      <source>(900,2100,100,100)</source>
      <offset>(-50,-5)</offset>
    </sprite>
    <average>(175,107,67,255)</average>
  </terrain>

**Comment by jkachhad** [comment]
I don't think this is the correct fix. Will have to check into it. The previous investigation suggested changing from Up to ClimbUp, but the fix create hacks accepting Up.
